sound/hybrid_font.h

```diff
@@ -173,11 +204,11 @@ class HybridFont : public SaberBase {
 	  drag.files_found()) {
 	PlayMonophonic(&drag, &drag);
       } else if (lockup.files_found()) {
-        if (bgnlock.files_found()) {
-          PlayMonophonic(&bgnlock, &lockup);
-        } else {
+	if (bgnlock.files_found()) {
```

Comment:
why change indentation here?
(and lots of places below)

Reply:
still fighting whitespace issues.  Thought I had them all resolved.

Reply:
tried my best, still getting weird whitespace issues.